src/test/java/org/apache/sling/resourceresolver/impl/mapping/VanityPathMapEntriesTest.java

```diff
@@ -1251,33 +1251,53 @@ public void test_bg_init_fallback_while_not_ready() {
             }
         });
 
-        assertEquals("test not started, query counter should be 0, queries were: " +
-                dumpQueries(queries), 0, queries.size());
+        Iterator<MapEntry> mit;
+        int expectedQueryCount = 0;
+        int expectedCacheHits = 0;
+        int expectedCacheMisses = 0;
 
-        mapEntries.vph.initializeVanityPaths();
-
-        assertEquals("test started, but query is blocked, query counter should still be 0, queries were: " +
-                dumpQueries(queries), 0, queries.size());
+        checkCounters("before tests",
+                queries, expectedQueryCount, expectedCacheHits, expectedCacheMisses);
 
-        // should not be finished until unlocked
-        assertFalse("VPH should not be ready as it is locked", mapEntries.vph.isReady());
+        mapEntries.vph.initializeVanityPaths();
+        assertFalse("VPH should not be ready until unblocked", mapEntries.vph.isReady());
+        checkCounters("after launch background it",
```

Comment:
is ``it`` an abbreviation (iterator) ? Or is just a left-over?

Reply:
typo